camus-sweeper/src/main/java/com/linkedin/camus/sweeper/CamusHourlySweeper.java

```diff
@@ -142,6 +142,10 @@ protected void runCollectorForTopicDir(FileSystem fs, String topic, Path topicSo
     return executorService.submit(new KafkaCollectorRunner(jobName, props, errorMessages, topic));
   }
 
+  private static void createTimeStampFileInFolder(FileSystem fs, Path folder, long timeStamp) throws IOException {
+    fs.createNewFile(new Path(folder, "_" + timeStamp));
+  }
+
   private class KafkaCollectorRunner extends CamusSweeper.KafkaCollectorRunner {
```

Comment:
It has since been renamed to CamusSingleFolderSweeper. I'll resolve conflict after review.